releasenotes/notes/only-rate-limit-on-sampling-rules-6f807defa7d0f5f1.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    tracing: Ensures ``DD_TRACE_RATE_LIMIT`` environment variable is only applied when tracer sampling is configured. By default rate limits should be applied by the Datadog Agent.
```

```suggestion
    tracing: Ensures ``DD_TRACE_RATE_LIMIT`` environment variable is only applied to spans for which tracer sampling is configured. For spans not matching sampling rules default rate limits should be applied by the Datadog Agent.
```
Not necessarily this exact modifcation, but I think that mentioning a bit more precisely how this change affects sampling for users would be helpful.